src/parsers/sany.ts

```diff
@@ -16,18 +17,21 @@ enum OutBlockType {
 export class SanyData {
     readonly dCollection = new DCollection();
     readonly modulePaths = new Map<string, string>();
+    readonly filePathToMonolithFilePath = new Map<string, string>();
 }
 
 /**
  * Parses stdout of TLA+ code parser.
  */
 export class SanyStdoutParser extends ProcessOutputHandler<SanyData> {
     rootModulePath: string | undefined;
+    monolithFilePath: string | undefined;
     curFilePath: string | undefined;
     outBlockType = OutBlockType.Parsing;
     errRange: vscode.Range | undefined;
     errMessage: string | undefined;
     pendingAbortMessage = false;
+    public getFileContents = (filePath : string) => readFileSync(filePath).toString(); // this should be set only at tests
```

Comment:
I've added this instead of use something like `sinon`, tell me if you know a better way to test it, I'm not very aware of the js/ts testing practices.